modxcms/revolution
# Fix missing labels and descriptions in TV quick create/edit window

### What does it do?
Made changes to address referenced issue and to improve the layout of the quick create/edit TV window:
- Moved select TV entries to the default topic to make them available application-wide 
- Updated SCSS to match field styles more closely with the main editing panel
- Changed the window mode to modal and darkened the overlay to better separate the window from the main view 

Below is a screenshot of how the window should appear:

<img width="1417" alt="Screen Shot 2021-12-28 at 9 20 41 PM" src="https://user-images.githubusercontent.com/689075/147621410-78228cb3-51c1-4e34-bfa9-ed30fc3c068d.png">

### Why is it needed?
There are missing labels and descriptions in the quick create/edit (QCE) form when working with a TV (when the main view is anything but the TV panel, as that's the only place where the TV-specific lexicons are loaded). Also, changing the window to a modal allows better focus on the task at hand.

### How to Test
1. From your terminal app, run `grunt build` from within the `_build/templates/default` folder and clear your browser cache.
2. Create at least one TV using the contextual quick create feature. Then, use quick edit on that TV (or others you may have created in your test environment). Try the QCE feature from different pages within the manager to verify labelling and descriptions remain visible.

### Special Note
Ultimately, more work will need to be done in a separate PR to bring the tv type-specific field display of the main TV panel to the associated QCE window (for the _default value_ and _input options_ fields). Also, I am aware that changing the window to a modal prevents the ability to drag/drop tree items into the window's fields. IMO, that's not particularly useful in the QCE windows and is problematic from my testing; it's very easy to unintentionally drop an item in the form beneath without knowing it. I've also found it very distracting to have the front window and the main page visually on the same plane. 

### Related issue(s)/PR(s)
Resolves #15947
